src/main/java/Storage.java

```diff
@@ -0,0 +1,104 @@
+import Tasks.*;
+import java.io.*;
+import java.util.*;
+
+/**
+ * Include all the file operations such as loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+    protected String filePath;
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Find the file according to the given filepath. If the file has not been created, create the file/
+     * @param filePath
+     */
+    public static void create(String filePath){
+        try {
+            File data = new File(filePath);
+            if (data.createNewFile()) {
+                System.out.println("No data file detected, I have created a new data file for you!");
+            } else {
+                System.out.println("I have found the file and loading the data for you!");
+            }
+        } catch (IOException e) {
+            System.out.println("An error occurred during file searching.");
+        }
+    }
+
+    /**
+     * Save the information into the file.
+     * @param textToAdd Texts going to be saved in the file.
+     * @param filePath
+     * @throws IOException
+     */
+
+    public static void save(String textToAdd,String filePath) throws IOException {
+        FileWriter fw = new FileWriter(filePath,false);
```

Comment:
Maybe you should consider a more descriptive name for the object like fileWriter.